docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+    "sequence": "12"
+  }
+}
+```
+
+The return data will contain the following fields:
+- `height` - the height at which the query was executed. This is currently not
+  supported and is `0` by default.
+- `data` - contains the result of the query.
+
+The `data` field returns a `BaseAccount`, which is the main struct used in TM2 to
+hold account data. It contains the following information:
+- `address` - the address of the account
+- `coins` - the list of coins the account owns
+- `public_key` - the TM2 public key of the account, which the address is derived from
+- `account_number` - a unique identifier for the account on the Gno.land chain
+- `sequence` - a nonce, used for protection against replay attacks
+
+### `bank/balances`
+
+With this query, we can fetch balances of a specific account. To call it, we can
+run the following command:
+
+```bash
+gnokey query bank/balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 -remote https://rpc.gno.land:443
+```
+
+If everything went correctly, we should get the following
+output:
+
+```bash
+height: 0
+data: "227984898927ugnot"
+```
+
+The data field will contain the coins the address owns.
+
+### `vm/qfuncs`
+
+Using the `vm/qfuncs` query, we can fetch exported functions from a specific package
+path. To specify the path we want to query, we can use the `-data` flag:
+
+```bash
+gnokey query vm/qfuncs --data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the `wugnot` realm:
+
+```json
+height: 0
+data: [
+        {
+          "FuncName": "Deposit",
+          "Params": null,
+          "Results": null
+        },
+        {
+          "FuncName": "Withdraw",
+          "Params": [
+            {
+            "Name": "amount",
+            "Type": "uint64",
+            "Value": ""
+            }
+          ],
+          "Results": null
+        },
+        // other functions
+]
+```
+
+### `vm/qfile`
+
+With the `vm/qfile` query, we can fetch files found on a specific package path.
+To specify the path we want to query, we can use the `-data` flag:
+
+```bash
+gnokey query vm/qfile -data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the
+`wugnot` realm:
+
+```bash
+height: 0
+data: gno.mod
+wugnot.gno
+z0_filetest.gno
+```
+
+### `vm/qeval`
+
+`vm/qeval` allows us to evaluate a call to an exported function without using gas,
+in read-only mode. For example:
+
+```bash
+gnokey query vm/qeval -remote https://rpc.gno.land:443 -data "gno.land/r/demo/wugnot
+BalanceOf(\"g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5\")" 
+```
+
+This command will return the `wugnot` balance of the above address without using gas.
+Properly escaping quotation marks, and inputting a new line for the function
+is currently required.
+
+### `vm/qrender`
+
+`vm/qrender` is an alias for executing `vm/qeval` on the `Render("")` function.
+We can use it like this:
```

Comment:
We should have something here also specifying that the package path and path argument to the `Render` function need to be specified by a newline character

Reply:
This has now been replaced with the . syntax; I've replaced the text but I don't remember the exact commit, please take a look at the file and resolve this convo if it fixes your comment :)